test/e2e/virtual/initializingworkspaces/virtualworkspace_test.go

```diff
@@ -240,11 +245,12 @@ func TestInitializingWorkspacesVirtualWorkspaceAccess(t *testing.T) {
 	}
 
 	t.Log("Set up RBAC to allow future calls to succeed")
+	scopedCtx := kcp.WithCluster(ctx, clusterName)
```

Comment:
these are not good. Keep them explicit. Don't alias. It harms readability of code as this aliasing is never uniform but always one-offs.